File: engine/ai/medic.lua
local Medic = setmetatable({}, ai.NPC)
Medic.__index = Medic

function Medic:new(x, y)
   return ai.NPC.new(self, x, y)
end

-------------
-- PRIVATE --
-------------

function Medic:__tostring()
   return '[Medic]'
end

function Medic:type() return 'Medic' end

return Medic

-- vim: ts=3:sw=3:sts=3:expandtab

File: engine/npc/medic.cc
#include "engine/npc/medic.h"

#include <memory>
using namespace std;

#include "./main.h"
#include "ui/ui.h"
#include "ui/dialogmanager.h"

Medic::Medic(Point init, class City& city)
    : Worker(init, city, 0)
{
}

void
Medic::TalkToHero()
{
}


// vim: ts=4:sw=4:sts=4:expandtab
